app/controllers/entries_controller.rb

```diff
@@ -31,16 +31,8 @@ def recover_entries_history(register_id, fields, params, page_size = 100)
     page = (params[:page] ||= 1).to_i
```

Comment:
this seems to be better as `page = (params[:page] || 1).to_i` or more my taste `page = params.fetch(:page) { 1 }.to_i` so you are expressing your intention that you want to access `:page` and want to use `1` as the default value instead of `nil`.